synchronization/src/main/java/de/cyface/synchronization/HttpConnection.java

```diff
@@ -70,10 +70,14 @@
      * The boundary to be used in the Multipart request to separate data.
      */
     final static String BOUNDARY = "---------------------------boundary";
+    /**
+     * The string which is used for a line feed.
+     */
+    final static String LINE_FEED = "\r\n";
```

Comment:
for better readability I replaced all occurrences with the variable. This helped me to find a bug in the boundary between the two file parts (I failed to notice that the file body needs to come directly after the multipart header of the file part.